_includes/integrations/react-tech-ref.md

```diff
@@ -284,40 +464,177 @@ Use the `tinymceScriptSrc` prop to specify an external version of TinyMCE to laz
 ```
 
 #### `value`
-This property allows the editor to be used as a controlled component by setting the `value` property and using the `onEditorChange` event.
+The HTML content of the editor when operating as a controlled component. 
+
+When this property is different to the current editor content, the editor content
+will be changed to match (within 200 milliseconds) and a undo level will be created.
+When the editor content changes by this mechanism an attempt will be made to
+retain the selection however if the previous selection does not exist in the new
+content the cursor will be returned to the start of the document.
```

Comment:
```suggestion
content, the cursor will be returned to the start of the document.
```

Reply:
With the comma before "however" I didn't like one here.